kalite/updates/views.py

```diff
@@ -32,7 +34,7 @@ def update_videos(request, max_to_show=4):
     if getattr(settings, 'USING_RASPBERRY_PI', False):
         messages.warning(request, _('For low-powered devices like the Raspberry Pi, please download less than 25 videos at a time.'))
 
-    if DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP:
+    if DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP or sys.platform == 'darwin':
```

Comment:
Sorry, didn't see this before now.

Why not just add a custom settings module for the mac installer and have `DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP = True` ?